primitives/xcm/src/ethereum_xcm.rs

```diff
@@ -296,68 +288,60 @@ mod tests {
 		};
 
 		let xcm_transaction = EthereumXcmTransactionV1 {
-			gas_limit: U256::from(1),
+			gas_limit: U256::one(),
 			fee_payment: EthereumXcmFee::Manual(ManualEthereumXcmFee {
-				gas_price: Some(U256::from(1)),
+				gas_price: Some(U256::one()),
 				max_fee_per_gas: None,
 			}),
 			action: TransactionAction::Call(H160::default()),
-			value: U256::from(0),
+			value: U256::zero(),
 			input: vec![1u8],
 			access_list: access_list.clone(),
 		};
 
-		let nonce = U256::from(0);
-		let gas_price = U256::from(1);
+		let nonce = U256::zero();
 		let expected_tx = Some(TransactionV2::EIP2930(EIP2930Transaction {
 			chain_id: 0,
 			nonce,
-			gas_price,
-			gas_limit: U256::from(1),
+			gas_price: U256::zero(),
```

Comment:
```suggestion
			gas_price: U256::one(),
```
ignore if this doesn't matter, just don't see why this changed from U256::from(1)